index.html

```diff
@@ -97,7 +95,7 @@ <h2>
             </ul>
           </div>
           <div class="project">
-            <button><a href="#">See project</a></button>
+            <button><a onclick="showPopUp(1)">See project</a></button>
           </div>
```

Comment:
- [ ] you should remove all the project section from html and add an array of data containing information about projects  like 
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
and loop through it and create the projects section dynamically with javascript

Reply:
is this a requirement? @youmari

Reply:
Yes, @gersonhiggins

Reply:
@joskalenda thanks